src/main/java/Deadlines.java

```diff
@@ -0,0 +1,14 @@
+public class Deadlines extends Task {
+
+    private String by;
```

Comment:
Similiar mistake as me! We can change the variable name 'by' to be more intuitive

Reply:
Since this class indicates a singular Deadline Task object, it could be better if you named your class in the singular form instead of the plural form. This also reduces confusion should the need for having a Deadline array arise.